docs/chap-networking.md

```diff
@@ -71,12 +71,12 @@ $$
 
 - ${b}_{{2}}$ and ${b}_{{3}}$ are a protobuf encoded field-value pair [indicating the used key type](https://github.com/libp2p/specs/blob/master/peer-ids/peer-ids.md#keys) (field ${1}$ of value ${1}$ implies *ED25519*).
 
-- ${b}_{{4}}$, ${b}_{{5}}$ and ${b}_{{{6}.{.37}}}$ are a protobuf encoded field-value pair where ${b}_{{5}}$ indicates the length of the public key followed by the the raw ED25519 public key itself, which varies for each Polkadot Host and is always 32 bytes (field ${2}$ contains the public key, which has a field value length prefix).
+- ${b}_{{4}}$, ${b}_{{5}}$ and ${b}_{{{6}.{.37}}}$ are a protobuf encoded field-value pair where ${b}_{{5}}$ indicates the length of the public key followed by the raw ED25519 public key itself, which varies for each Polkadot Host and is always 32 bytes (field ${2}$ contains the public key, which has a field value length prefix).
 
 :::
-## -sec-num- Discovery mechanism {#sect-discovery-mechanism}
+## -sec-num- Network bootstrap and discovery {#sect-network-bootstrap}
```

Comment:
Bootstrapping is nothing more but "discovery but at the very beginning". There IMO should be no need to mention the word "bootstrap" anywhere in the spec, it just adds more vocabulary for no gain.

Reply:
The choice for " Network bootstrap and discovery" was made to distinguish "Authority discovery" from the "discovery process" to bootstrap the network. Actually, the bootstrap (or bootstrapping process) is a well-defined and accepted definition in the p2p literature and has been used for decades. e,g.,:

- "Bootstrapping of Peer-to-Peer Networks," C. Gauthier Dickey and C. Grothoff, -  International Symposium on Applications and the Internet 2008
- 
- "Bootstrapping large-scale DHT networks." Jae Woo Lee, Henning Schulzrinne, Wolfgang Kellerer, and Zoran Despotovic. 2007. In Proceedings of the 2007 ACM CoNEXT conference 

- "Bootstrapping locality-aware P2P networks," C. Cramer, K. Kutzner and T. Fuhrmann, , Proceedings. 2004 12th IEEE International Conference on Networks 2004